website/blog/2022-04-08-adding-fonts.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Adding fonts
+description: An intro to adding fonts to your project.
+slug: adding-fonts
+authors: iobrado
+date: 2022-04-12
+tags: [eightshift, boilerplate, fonts]
+hide_table_of_contents: false
+---
+
+Every project is unique. Logo, colors, fonts, etc. are what define the visual identity of your website. In this post, we'll cover adding fonts to a project.
+<!--truncate-->
+
+## Importing fonts into your project
+[Our documentation](/docs/basics/fonts) covers the necessary steps to add a font to your project, but here we'll cover the process in a bit more detail. To start, we need a font (or two). For this example, I'll use **_Source Sans Pro_** and **_Noto Serif_** which I've downloaded from [Google Fonts](https://fonts.google.com/). They are in the `.ttf` format (you may find some which are `.otf`, which will work in the same way), so you need to convert them to `.woff` and `.woff2`. Whatever approach you use is okay - we recommend the following tools:
```

Comment:
I'd add a note at the end that says:

> Just make sure you have the proper license for the fonts you are converting.

I think that [font squirrel](https://www.fontsquirrel.com/) would even prevent you from converting fonts that are premium.